jdaviz/app.py

```diff
@@ -939,16 +941,23 @@ def get_subsets(self, subset_name=None, spectral_only=False,
             Return only object relevant information and
             leave out the region class name and glue_state.
         simplify_spectral : bool
-            Return a composite spectral subset collapsed to a simplified SpectralRegion.
+            Return a composite spectral subset collapsed to a simplified
+            SpectralRegion.roi_subset_state_to_region
         use_display_units: bool, optional
```

Comment:
While you are at it, should be a space here.
```suggestion
        use_display_units : bool, optional
```